privacy-policy/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: page
+title: Privacy Policy
+---
+
+# Vox Pupuli Privacy Policy
+
+This document details the data the Vox Pupuli collects about you, why it is
+collected, what is done with it, and your options for how it's processed.
+
+The Vox Pupuli project does not share data with 3rd parties, except for hosting
+requirements (e.g. GitHub, YouTube) as detailed below, or when required to by
+law.
+
+# Types of data we collect
+
+* Contact details
+  * Name
+  * E-Mail address
+
+# How and why we use your data
+
+## The website
+
+The Vox Pupuli website is a static site that does not track, collect, or store 
+infromation or set cookies. There is no account needed or required to access 
+the website itself. The website itself is hosted with [Github Pages](https://pages.github.com/) 
+and any data gathered by Github will adhere to the [Github Privacy Policy](https://help.github.com/articles/github-privacy-statement/).
+
+The website does host a community run Blog series and do require that the 
+author's name be visible in the blog post indefinitely.
+
+## The codebases
+
+The Vox Pupuli community uses [GitHub](https://github.com) to manage the code 
+that comprises the various Puppet modules, plugins, tools, and RubyGems the 
+community is responsible for maintaining. Refer to [GitHub's Privacy Policy](https://help.github.com/articles/github-privacy-statement/)
+for information on how GitHub handles the data you post there. We also use 
+non-identifiable data from the codebases to provide aggregated statistics such 
+as total member count, number of open PRs, total number of projects, etc.
```

Comment:
there are a few third party tools that interact with our codebase
![2018-05-27-085350_734x576_scrot](https://user-images.githubusercontent.com/341866/40583270-89b80c1c-618b-11e8-9baf-b89bf11e483e.png)

We probably need to review which data we share with them?

Reply:
We probably need to review those in general. Gemnasium, for example, is now owned by Gitlab and all their functionality has been rolled into Gitlab.com and Self-Hosted Gitlab. No idea what ZenHub or zube are.